third_party/ienn/src/ie_compilation.cpp

```diff
@@ -122,8 +117,9 @@ int32_t Compilation::Compile() {
   }
 
   try {
-    network_.reset(
-        new CNNNetwork(Builder::convertToICNNNetwork(builder_->build())));
+    ngraph_function_ = std::make_shared<Function>(outputs_, inputs_);
+    network_.reset(new CNNNetwork(ngraph_function_));
+    std::cout << "[IE] succeed to creat ngraph" << std::endl;
```

Comment:
Remove unused `std::cout`

Reply:
Use local variables for `outputs_ `and `inputs_`

Reply:
> Use local variables for `outputs_ `and `inputs_`

Hi, `outputs_ `and `inputs_ ` are also needed in `AddInput(uint32_t index)` and `AddOutput(uint32_t index)`

Reply:
Please use local variable if it's only used in one function except unimpementation.